src/components/pages/Map/MapPage.jsx

```diff
@@ -39,8 +40,18 @@ export default class MapPage extends React.Component {
     return (
       <div className="c-map-page l-map-page">
         <Sidebar setSidebarWidth={() => {}}>
-          <h1>Sidebar</h1>
-          <div>Content</div>
+          <SegmentedUi
+            className="-tabs"
+            items={[{ label: 'map view', value: 'mapView' }, { label: 'analyse locations', value: 'analyseLocations' }]}
```

Comment:
I don't think that putting the tab names inside a constant would be the right approach. It isn't a list of a select values, directly related with some backend values; it's just some ui literales. Would you move them to a constant file if it just were a simple Boostrap tabs?